stock_picking_completion_info/models/stock_picking.py

```diff
@@ -0,0 +1,59 @@
+# Copyright 2019 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+from odoo import api, models, fields
+
+
+class PickingType(models.Model):
+
+    _inherit = 'stock.picking.type'
+
+    display_completion_info = fields.Boolean(
+        help='Inform operator of a completed order pick at processing and at'
+             ' completion'
+    )
+
+
+class StockPicking(models.Model):
+
+    _inherit = 'stock.picking'
+
+    completion_info = fields.Selection(
+        [
+            ('no', 'No'),
+            ('last_picking', 'Completion of this picking allows next pickings '
+                             'to be processed.'),
+            ('next_picking_ready', 'Next pickings are ready to be processed.')
```

Comment:
```suggestion
            ('next_picking_ready', 'Next operations are ready to be processed.')
```
Sames for here, Would it be feasible to provide a hyperlink on the next operations ?